src/expanses/expanses.interface.ts

```diff
@@ -0,0 +1,11 @@
+interface Expanse {
```

Comment:
maybe u can use `CreateExpanseDto` as a type🤔

Reply:
CreateExpanse should be without **id** (will fix it in next commit) so i can validate incoming JSON to not contain any external keys, and to not allow duplicate id which is generated on "server"

Reply:
Same for User interface and User Dto

Reply:
u can use something looks like this 

```
type Expanse = CreateExpanseDto & {
  id: number
};

Reply:
or create a utility for this issue

type Expanse = ModelWithId<CreateExpanseDto>